full.md

```diff
@@ -32,10 +32,16 @@ layout: lipu
 
 * [kule epiku Atawan](https://lipu-pi-ijo-pi-toki.pona.la)
 
+* [jan Janpa](https://janpa.li.pona.la)
+
 ### Defunct subdomains
 
 * kulupu.pona.la - forum by soweli Masi. Closed [due to moderation concerns](https://web.archive.org/web/20231114030939/https://kulupu.pona.la/d/155-notice-of-the-shutdown-of-kulupu-lipu-pona-until-further-notice), but [is available as an archive](https://archive.org/details/kulupu.pona.la).
 
 * linku.pona.la - use [linku.la](https://linku.la) instead
 
 * lipu.pona.la - Writefreely instance by jan Kita. Closed due to spam abuse.
+
+* toki.pona.la - see kulupu.pona.la (previously used subdomain before choosing kulupu.pona.la)

```

Comment:
it's unhelpful to specify this, there should be a different place to keep documentation of unused pages